tests/unit/testsuite/Bolt/Boltpay/Helper/ConfigTraitTest.php

```diff
@@ -1,66 +1,946 @@
+                    'active' => '',
+                    'everywhere' => ''
+                )
+            ),
+            array(
+                'case' => array(
+                    'expect' => false,
+                    'active' => true,
+                    'everywhere' => false
+                )
+            ),
+            array(
+                'case' => array(
+                    'expect' => false,
+                    'active' => false,
+                    'everywhere' => true
+                )
+            ),
+            array(
+                'case' => array(
+                    'expect' => true,
+                    'active' => true,
+                    'everywhere' => true
+                )
+            ),
+            
+        );
+    }
+
+    /**
+     * @test
+     * @group Trait
+     * @dataProvider getApiKeyConfigCases
+     * @param array $case
+     */
+    public function getApiKeyConfig(array $case)
+    {
+        $this->app->getStore()->setConfig('payment/boltpay/extra_options', $case['extra_options']);
+        $result = $this->mock->getApiKeyConfig();
+        $this->assertInternalType('string', $result);
+        $this->assertEquals($case['expect'], $result);
+    }
+
+    /**
+     * Test cases
+     * @return array
+     */
+    public function getApiKeyConfigCases()
+    {
+        return array(
+            array(
+                'case' => array(
+                    // Looks like default value
+                    'expect' => '66d80ae8d0278e3ee2d23e65649b7256',
+                    'extra_options' => ''
+                )
+            ),
+            array(
+                'case' => array(
+                    'expect' => 'test.key',
+                    'extra_options' => '{"datadogKey": "test.key"}'
+                )
+            ),
+            
+        );
+    }
+
+    /**
+     * @test
+     * @group Trait
+     * @dataProvider getSeverityConfigCases
+     * @param array $case
+     */
+    public function getSeverityConfig(array $case)
+    {
+        $this->app->getStore()->setConfig('payment/boltpay/extra_options', $case['extra_options']);
+        $result = $this->mock->getSeverityConfig();
+        $this->assertInternalType('array', $result);
+        $this->assertEquals($case['expect'], $result);
+    }
+
+    /**
+     * Test cases
+     * @return array
+     */
+    public function getSeverityConfigCases()
+    {
+        return array(
+            array(
+                'case' => array(
+                    'expect' => array(''),
+                    'extra_options' => '{"datadogKeySeverity": ""}'
+                )
+            ),
+            array(
+                'case' => array(
+                    'expect' => array('1', '2', '5', '6'),
+                    'extra_options' => '{"datadogKeySeverity": "1,2,              5                , 6"}'
+                )
+            ),
+        );
+    }
+
+    public function tearDown()
+    {
+        //Mage::reset();
```

Comment:
remove?

Reply:
@daisy1754 Yes!!!
This line crushed unit tests chain with
PHP Fatal error:  Cannot redeclare benchmark() (previously declared in /home/aymelyanov/www/magento-one/app/code/community/Bolt/Boltpay/Model/Observer.php:48)